src/lubed/gh.py

```diff
@@ -1,8 +1,74 @@
 """Interact with GitHub and create issues"""
 # SPDX-License-Identifier: GPL-3.0-or-later
-from typing import List
+from typing import Any, Dict, List
 import github
 
+from gql import gql, Client
+from gql.transport.aiohttp import AIOHTTPTransport
+
+
+def assign_issue_to_board(
+    issue_id: str,
+    board_id: str,
+    gh_token: str,
+) -> Dict[str, Any]:
+    """Assign issue to a project board.
+
+    :param issue_id: Issue Node ID.
+    :param board_id: Github Board (V2) Node ID.
+    :param gh_token: GitHub OAuth token.
+    :returns: GraphQL execution result.
+    """
+    client = _get_gql_client(gh_token)
+    link_issue_project_gql = gql(
+        """
+    mutation($issue_id: ID!, $board_id: ID!) {
+      addProjectV2ItemById(input:{contentId:$issue_id,projectId:$board_id}) {
+        item {
+          id
+        }
+      }
+    }
+    """
+    )
+    return client.execute(
+        link_issue_project_gql,
+        variable_values={
+            "issue_id": issue_id,
+            "board_id": board_id,
+        },
+    )
+
+
+def get_issue_node_id(issue_num: int, repository: str, gh_token: str) -> str:
```

Comment:
[nitpick]: for consistency, I'd use the same argument name `repo_name` that's used in `create_issue_in_board`.